autogen/agentchat/contrib/retrieve_user_proxy_agent.py

```diff
@@ -128,6 +128,9 @@ def __init__(
                 - update_context (Optional, bool): if False, will not apply `Update Context` for interactive retrieval. Default is True.
                 - get_or_create (Optional, bool): if True, will create/recreate a collection for the retrieve chat.
                     This is the same as that used in chromadb. Default is False.
+                - custom_token_count_function(Optional, Callable): a custom function to count the number of tokens in a string.
+                    The function should take a string as input and return three integers (token_count, tokens_per_message, tokens_per_name).
+                    Default is None, tiktoken will be used and works well for OpenAI models, but may not accurate for other models.
```

Comment:
```suggestion
                - custom_token_count_function (Optional, Callable): a custom function to count the number of tokens in a string.
                    The function should take a string as input and return three integers (token_count, tokens_per_message, tokens_per_name).
                    Default is None, tiktoken will be used and works well for OpenAI models, but may not be accurate for other models.
```

Reply:
Further suggestion for the last line:
`
If None (the default) use tiktoken, which may not be accurate for non-OpenAI models.
`